dart/lib/src/event_processor/enricher/io_enricher_event_processor.dart

```diff
@@ -40,6 +40,11 @@ class IoEnricherEventProcessor implements EnricherEventProcessor {
     );
 
     contexts['dart_context'] = _getDartContext();
+    contexts['process_info'] = <String, dynamic>{
+      'currentResidentSetSize':
```

Comment:
let's add currentRss to app_memory

Reply:
How does `currentRss` compare to what the native side sets? Does it override what the native side sets?

Reply:
We only set this if there are no native SDKs

Reply:
I don't think android sets it, I know iOS for sure does @denrase

Reply:
Updated so we set `appMemory` to `ProcessInfo.currentRss`, it it's not already set, regardless of native integration. The other values are merged, so this is additional information.

On Linux/Windows we now also add free and total system memory, if possible.